src/dex-ui/routes/router.tsx

```diff
@@ -38,17 +41,17 @@ export const router = createBrowserRouter(
       <Route path={`${Paths.Pools.AddLiquidity}/:pairId/:poolName`} element={<AddLiquidityPage />} />
       <Route path={Paths.Pools.Withdraw} element={<WithdrawPage />} />
       <Route path={Paths.Pools.CreatePool} element={<CreatePoolPage />} />
-      {/* TODO: Create governance route heirarchy. */}
-      <Route path={Paths.Governance.default} element={<Governance />} />
-      <Route path={`${Paths.Governance.ProposalDetails}/:id`} element={<ProposalDetails />} />
-      <Route path={Paths.Governance.SelectProposalType} element={<SelectProposalType />} />
-      <Route path={Paths.Governance.CreateNewToken} element={<CreateProposal proposalType="new-token" />} />
-      <Route path={Paths.Governance.CreateText} element={<CreateProposal proposalType="text" />} />
-      <Route path={Paths.Governance.CreateTokenTransfer} element={<CreateProposal proposalType="token-transfer" />} />
-      <Route
-        path={Paths.Governance.CreateContractUpgrade}
-        element={<CreateProposal proposalType="contract-upgrade" />}
-      />
+      <Route path={Paths.Governance.default}>
+        <Route index element={<Governance />} />
+        <Route path={`${Paths.Governance.ProposalDetails}/:id`} element={<ProposalDetails />} />
+        <Route path={Paths.Governance.SelectProposalType} element={<Outlet />}>
```

Comment:
I think we should change the `select-proposal-type` URL path to `create-new-proposal`. The new naming matches the title and purpose of the page and is more semantically correct in the child urls that follow. For example, `create-new-proposal/text` more accurately describes what the user is trying to do than `select-proposal-type/text`.

Reply:
Done